code/src/components/app-input/AppInput.scss

```diff
@@ -0,0 +1,12 @@
+@import "@design-system";
+
+.spa-app-input {
+  &__textfield {
+    .css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input {
+      padding: 13.5px 11px;
+    }
+    .css-nxo287-MuiInputBase-input-MuiOutlinedInput-input {
+      padding: 13.5px 11px;
+    }
+  }
```

Comment:
Don't use `css-1t8l2tu` css class like css-blablabla, because it is autogenerated and we can get wrong selector in the future. Also consider using `spacing` here

Reply:
Fixed